src/resources/Alteration/BC.ts

```diff
@@ -51,5 +51,6 @@ export const AlterationResourceBc: ResourceIF = {
   },
   certifyClause: 'Note: It is an offence to make a false or misleading statement in respect ' +
     'of a material fact in a record submitted to the Corporate Registry for filing. ' +
-    'See section 427 of the Business Corporations Act.'
+    'See section 427 of the Business Corporations Act.',
+  certifyText: 'business'
```

Comment:
Does every resource file now provide `certifyText`? If so then please change it to non-optional in the interface (resource-interface.ts).

Reply:
I just checked. SP and GP don't use it, so it needs to remain optional. No change needed. Resolving.